Backpack/Label/Classes/BPKLabel.m

```diff
@@ -100,11 +101,24 @@ - (void)setFontMapping:(BPKFontMapping *_Nullable)fontMapping {
     }
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *_Nullable)previousTraitCollection {
+    [super traitCollectionDidChange:previousTraitCollection];
+
+#if __BPK_COMPILING_WITH_DARK_MODE
+    if (@available(iOS 13.0, *)) {
+        UITraitCollection *traitCollection = [UITraitCollection currentTraitCollection];
+        if ([previousTraitCollection hasDifferentColorAppearanceComparedToTraitCollection:traitCollection]) {
+            [self updateTextStyle];
+        }
+    }
+#endif
+}
+
 #pragma mark - Private
 
 - (void)setupWithStyle:(BPKFontStyle)style {
     self.fontStyle = style;
-    self.textColor = BPKColor.gray700;
+    self.textColor = BPKColor.label;
```

Comment:
If the consumer has set the colour themselves then we should respect that here instead of changing to the default colour for the current appearance...